test/parallel/test-err-name-deprecation.js

```diff
@@ -1,16 +1,9 @@
 'use strict';
 const common = require('../common');
 
-// Flags: --pending-deprecation
-
-common.expectWarning({
-  DeprecationWarning: [
-    ['process.binding() is deprecated. Please use public APIs instead.',
```

Comment:
It seems that this is the only place where DEP0111 is  tested. Please add a new test for it with  `--pending-deprecation` on.

Reply:
Thanks for reviewing!

Fixed